pywwise/ak/wwise/core/transport.py

```diff
@@ -1,54 +1,154 @@
+        :return: True if the action was executed, False otherwise.
+        """
+        if transport_action is None:
+            return False
+        
+        args = {"action": transport_action, **({"transport": transport_id} if transport_id is not None else {})}
+        
+        return self._client.call("ak.wwise.core.transport.execute", args) is not None
+
+    def get_list(self) -> tuple[TransportObjectInfo, ...]:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_getlist.html \n
+        Returns the list of transport objects.
+        :return: A list of transport objects.
+        """
+        results = self._client.call("ak.wwise.core.transport.getList", {})
+        
+        results = results.get("list") if results is not None else None
+        
+        if results is None:
+            return ()
+        
+        transport_objects: list[TransportObjectInfo]
+        
+        for result in results:
+            object = GUID(result["object"])
+            game_object = int(result["gameObject"])
+            transport = int(result["transport"])
+            transport_objects.append(TransportObjectInfo(object, game_object, transport))
+        
+        return tuple(transport_objects)
+    
+    def get_state(self, transport_id: int) -> ETransportState:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_getstate.html \n
+        Gets the state of the given transport object.
+        :param transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: An enum value that represents the state of the transport object. Return value can either be: Playing,
+        Stopped, Paused, or None. In case its none, the method call failed.
+        """
+        if transport_id is None:
+            return ETransportState.NONE
+        
+        args = {"transport": transport_id}
+        
+        result = self._client.call("ak.wwise.core.transport.getState", args)
+        
+        if result is None:
+            return ETransportState.NONE
+        
+        message = result.get("state")
+        
+        return EnumStatics.from_value(ETransportState, result.get("state", "None"))
+        # from_value will [attempt to] convert any value to the specified Enum subclass.
+    
+    def prepare(self, object: GUID | ProjectPath | tuple[EObjectType, Name]) -> bool:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_prepare.html \n
+        Prepare the object and its dependencies for playback. Use this function before calling
+        `PostEventSync` or `PostMIDIOnEventSync` from `IAkGlobalPluginContext`.
+        :param object: The name of the object qualified by its type or Short ID in the form of type:name or
```

Comment:
Incorrect documentation: we don't support ShortID here. Although the AK documentation states that, it is often contradictory (unfortunately).